designer/server/src/types.ts

```diff
@@ -28,7 +32,7 @@ declare module '@hapi/hapi' {
   interface Server {
     method(name: 'session.drop', method: SessionCache['drop']): void
     method(name: 'session.get', method: SessionCache['get']): void
-    method(name: 'session.set', method: SessionCache['set']): void
+    method(name: 'session.set', method: SessionCache['set'], ttl?: number): void
```

Comment:
Looks like we're not using the 3rd param, can you double check?

It's the `method` function that gets the TTL as the 3rd param

Reply:
you are right. Removed it.